oriedita-ui/src/main/java/oriedita/editor/swing/LeftPanel.java

```diff
@@ -366,6 +374,23 @@ public void popupMenuWillBecomeInvisible(PopupMenuEvent e) {
                 replace_lineButton.doClick();
             }
         });
+        switchReplaceButton.addActionListener(e -> {
```

Comment:
could you make this a separate action instead of an action listener? it would be nice to have it decoupled so we can move the button around freely without having to move lots of code as well.

Reply:
Pushed the new code, check it out.

Reply:
Hold on let me move that registerAction down a bit